seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/src/main/scala/org/apache/seatunnel/spark/clickhouse/sink/Table.scala

```diff
@@ -102,4 +105,10 @@ class Table(val name: String, val database: String, val engine: String, val crea
       CheckResult.success()
     }
   }
+
+  def supportReplicatedMergeTree(ddl: String): String = {
```

Comment:
`supportReplicatedMergeTree` replace to `localizationEngine` will be more clear. Other guys maybe can do more in this method.

Reply:
ok,i understand what you mean.i will change it.thank you for your advice.